buildSrc/src/main/kotlin/ApplicationConfig.kt

```diff
@@ -0,0 +1,10 @@
+object ApplicationConfig {
+    val id = "com.simtop.billionbeers"
+
+    const val compileSdk = 29
+    const val minSdk = 21
+    const val targetSdk = 29
+
+    const val versionCode = 1
+    const val versionName = "0.01"
```

Comment:
Each library had a different versionName, the plugin automatically wrote 0.01, but if we need to have different version name for each library we have to create the const val manually